static/gsApp/hooks/orgStatsProfilingBanner.tsx

```diff
@@ -0,0 +1,123 @@
+import styled from '@emotion/styled';
+import type {Location} from 'history';
+
+import {Button} from 'sentry/components/core/button';
+import {IconProfiling} from 'sentry/icons';
+import {space} from 'sentry/styles/space';
+import type {InjectedRouter} from 'sentry/types/legacyReactRouter';
+import normalizeUrl from 'sentry/utils/url/normalizeUrl';
+
+type Props = {
+  router: InjectedRouter & {location?: Location};
+};
+
+function OrgStatsProfilingBanner({router}: Props) {
+  return (
+    <BannerGrid>
+      <HeaderItem>
+        <h6>Profiling has a new billing model!</h6>
+        <span>
+          We've split Profiling into two products targeted at different use cases –
+          Continuous Profiling for the backend and UI Profiling for the frontend. These
+          products are billed separately.
+        </span>
+      </HeaderItem>
+      <CategoryItemLeft>
+        <h6>UI Profile Hours</h6>
+        <span>
+          Ensure great UX on browser and mobile apps by fixing issues that cause long load
+          times and unresponsive interactions.
+        </span>
+        <div>
+          <StyledProfilingButton
+            size="sm"
+            icon={<IconProfiling size="sm" />}
```

Comment:
Question: Do we have the "eye" icon in [the Figma](https://www.figma.com/design/IjYeQp4VI5vjoDBaWZTTcO/Continuous-profiling-edge-cases?node-id=150-511) available in the `sentry` repo? 

<img width="160" alt="Screenshot 2025-04-25 at 10 10 27 AM" src="https://github.com/user-attachments/assets/0a5de823-03c6-466f-b7e6-21fa3fcdc54a" />